src/styles/ParentStyles/index.less

```diff
@@ -142,8 +143,77 @@
   margin: 5%;
 }
 
+//****Parent-Progress styles****
+.progress-bars {
```

Comment:
I wonder if .progress-bars and .progress-bar can be combined into one, layered component, since they share some characteristics?